forc-client-ops/Cargo.toml

```
@@ -0,0 +1,23 @@
+[package]
+name = "forc-client-ops"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1"
+clap = { version = "3", features = ["derive"] }
+forc = {path = "../forc"}
+forc-pkg = {path = "../forc-pkg"}
+forc-util = {path = "../forc-util"}
```

Could you make the formatting of these dependencies consistent with the rest of the file? e.g.

```rust
forc-util = { path = "../forc-util" }
```

Also, for dependencies within the same workspace, we need to specify the version of these dependencies in order for `cargo publish` to work, e.g.

```suggestion
forc-util = { version = "0.19.1", path = "../forc-util" }
```